libse/SubtitleFormats/SubtitleFormat.cs

```diff
@@ -397,41 +397,39 @@ public virtual bool IsTextBased
             }
         }
 
-        protected static TimeCode DecodeTimeCodeFrames(string[] parts)
+        protected TimeCode DecodeTimeCodeFramesTwoParts(string[] parts)
         {
             if (parts == null)
                 return new TimeCode(0, 0, 0, 0);
+            if (parts.Length != 2)
+                throw new InvalidOperationException();
+            // 00:00
+            return new TimeCode(0, 0, int.Parse(parts[0]), FramesToMillisecondsMax999(int.Parse(parts[1])));
+        }
 
-            int hour = 0;
-            int minutes = 0;
-            int seconds = 0;
-            int frames = 0;
-
-            if (parts.Length == 4)
-            {
-                hour = int.Parse(parts[0]);
-                minutes = int.Parse(parts[1]);
-                seconds = int.Parse(parts[2]);
-                frames = int.Parse(parts[3]);
-            }
-            else if (parts.Length == 3)
-            {
-                minutes = int.Parse(parts[0]);
-                seconds = int.Parse(parts[1]);
-                frames = int.Parse(parts[2]);
-            }
-            else if (parts.Length == 2)
-            {
-                seconds = int.Parse(parts[0]);
-                frames = int.Parse(parts[1]);
-            }
+        protected TimeCode DecodeTimeCodeFramesThreeParts(string[] parts)

```

Comment:
Not used but I found a sub-format which filters parts e.g: `parts.Length >= 3`. after I ran test it still calls `DecodeTimeCodeFramesFourParts`